linux_os/guide/system/software/system-tools/package_python-rhsm_installed/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,rhel8,rhv4
```

Comment:
```
$ git grep -l python-rhsm
linux_os/guide/system/software/system-tools/package_python-rhsm_installed/rule.yml
rhel6/kickstart/ssg-rhel6-stig-ks.cfg
```

It seems like this rule has no OVAL in all products anyway.

Reply:
Yes, just description for the rules is there.